src/ddmd/mars.d

```diff
@@ -186,67 +187,6 @@ Where:
 ", FileName.canonicalName(global.inifilename), fpic, m32mscoff, mscrtlib);
 }
 
-/// DMD-generated module `__entrypoint` where the C main resides
-extern (C++) __gshared Module entrypoint = null;
-/// Module in which the D main is
-extern (C++) __gshared Module rootHasMain = null;
```

Comment:
I'll be happy when `grep 'import ddmd.mars;' ` returns no results, as that would unblock one of the stoppers for gdc/ddmd.  If this does just that then please link in the bugzilla report I created.